esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -607,6 +619,7 @@ define(
     SuperFiles: "Superfiles",
     SuperFilesBelongsTo: "Miembro de Super-Archivo(s)",
     SuperfilesOnly: "Solo Superfiles",
+    SuperOwner: "Dueño de Super-Archivos",
```

Comment:
line 621 uses the term "Superfiles" but we use "Super-Archivos" here and line 620, we should pick one.